packages/accounts/package.json

```diff
@@ -39,7 +39,6 @@
     "test:run-e2e": "vitest run --config vitest.config.e2e.ts"
   },
   "devDependencies": {
-    "@alchemy/aa-alchemy": "^1.0.0",
```

Comment:
removing b/c now alchemy has a dependency on aa-accounts, so this would be a circular dependency

I know this is a dev dependency so not critical, but trying to set good practices